### ISSUE TITLE
Expose optional features

### ISSUE THREAD

**Issue by bendemboski** [pzuraq/ember-compatibility-helpers#27]
There is currently no public API for accessing the state of optional features at runtime, and this addon seems like the ideal place. Something like:

```javascript
import { hasOptionalFeature } from 'ember-compatibility-helpers';

if (hasOptionalFeature('jquery-integration')) {
  // ...
}
```

I think this will either require, or else supercede https://github.com/emberjs/ember-optional-features/issues/10, but either way, should address https://github.com/emberjs/ember-optional-features/issues/10#issuecomment-421180959.

**Comment by rwjblue** [comment]
I totally agree.

**Comment by knownasilya** [comment]
Yeah, trying to write tests that work with classic and v4, and hitting this. Any ideas?
